src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/FixCaseApiService.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ecm.common.model.ccd.CCDRequest;
+import uk.gov.hmcts.ecm.common.model.ccd.CaseData;
+import uk.gov.hmcts.ecm.common.model.multiples.SubmitMultipleEvent;
+import uk.gov.hmcts.ethos.replacement.docmosis.helpers.MultiplesHelper;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.excel.MultipleCasesReadingService;
+
+import java.util.List;
+
+@RequiredArgsConstructor
+@Service("fixCaseApiService")
+public class FixCaseApiService {
+
+    private final MultipleCasesReadingService multipleCasesReadingService;
+
+    @Value("${ccd_gateway_base_url}")
+    private String ccdGatewayBaseUrl;
+
+    public void checkUpdateMultipleReference(CCDRequest ccdRequest, String userToken) {
+        CaseData caseData = ccdRequest.getCaseDetails().getCaseData();
+        if (caseData.getEcmCaseType().equals("Multiple")
+                && (caseData.getMultipleReference().length() == 16
```

Comment:
One thing to note Cindy is that we have 2 different fields to display multiple references
- multipleReference
- multipleReferenceLinkMarkUp

The multiple reference will always be a 7 digit number. It is the 'LinkMarkUp' that is displaying the 16 digit number. You can use [Case 1800629/2021](https://manage-case.demo.platform.hmcts.net/print/jurisdictions/EMPLOYMENT/case-types/Leeds/cases/1627459424482379) as an example. If you scroll down in the print, you can see it's the LinkMarkUp field that is 16 digits and not the multipleReference

Reply:
My approach would be that if the user has entered a multiple reference and the type is set to Multiple then just automatically update the markup field. Don't bother checking if it valid or not - just update it every time